doc/src/manual/ranch.set_transport_options.asciidoc

```diff
@@ -56,10 +61,14 @@ transport options contain invalid values.
 
 == Examples
 
-.Set the transport options
+.Update the listener TCP port within the `socket_opts` transport option
```

Comment:
Same here, should have separate examples.

Reply:
OK